Makefile

```
@@ -26,5 +26,8 @@ boot.firm:	$(SUBFOLDERS)
 	-A 0x18180000 -C XDMA XDMA NDMA XDMA
 	@echo built... $(notdir $@)
 
+boot.3dsx:
+	@curl -sS "https://github.com/fincs/new-hbmenu/releases/download/v2.1.1/boot.3dsx" -o "boot.3dsx"
```

Hmm, maybe always download the latest version instead of a specific version which might need to be changed in the future ?

```suggestion
	@curl -sS "https://github.com/fincs/new-hbmenu/releases/latest/download/boot.3dsx" -o "boot.3dsx"
```

---

Thank you for the suggestion! Still, it seems like we'll need to follow a redirect for GitHub to serve the actual .3dsx file, so I'll commit the suggestion and then add the missing switch.